amundsen_application/static/js/components/DashboardPage/QueryList/index.tsx

```diff
@@ -19,6 +20,7 @@ class QueryList extends React.Component<QueryListProps> {
     const queryList = queries.map(({ name, query_text, url }) => (
       <QueryListItem
         key={`key:${name}`}
```

Comment:
curious: what is ` here in the code?

Reply:
in order to access name? could we use ```key: {name} ``` syntax?

Reply:
Syntax for concatenating strings. See next comment posted a link to some JS doc.